docs/includes/_sdk-setup/code-snippets/initialize-swift.mdx

```diff
@@ -0,0 +1,29 @@
+```swift
+// --- Initialize with required arguments
+Embedded.shared.initialize(
+  biometricAskPrompt: "Please provide your biometric"
+) { result in
+  switch result {
+  case .success():
+    print("Initialization successful")
+  case .failure(let error):
+    print("Initialization failed: \(error)")
+  }
+}
+
+// --- Initialize with required and optional arguments
+Embedded.shared.initialize(
+  allowedDomains: ["example.com", "another-example.com"],
```

Comment:
we should remove allowedDomains from all the SDKs `initialize` and `getAuthenticationContext`